content/blog/its-about-time/index.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+output: hugodown::hugo_document
+
+slug: its-about-time
+title: "It's about time"
+date: 2022-09-28
+author: Mara Averick
+description: >
+    Davis Vaughan's talk from rstudio::conf(2022) on clock, an R package that aims to provide
+    comprehensive and safe handling of date-times.
+
+photo:
+  url: https://www.pexels.com/photo/collection-of-retro-wall-clocks-in-antique-store-2168241/
+  author: Teddy Yang
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [learn] 
+tags: [clock, r-lib]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+At rstudio::conf(2022), Davis Vaughan gave a lightning talk on [clock](https://clock.r-lib.org/), an R package that aims to provide comprehensive and safe handling of date-times. clock goes beyond the date and date-time types that base R provides, implementing new types for year-month, year-quarter, ISO year-week, and many other date-like formats, all with up to nanosecond precision.
```

Comment:
I think we need to say something about lubridate here: i.e. lubridate is not going away, and we expect to update lubridate in the future to use clock as a backend (so clock becomes stringi to lubridate's stringr)

Reply:
Done!